code/datums/emotes.dm

```diff
@@ -21,6 +21,7 @@
 	var/list/mob_type_ignore_stat_typecache
 	var/stat_allowed = CONSCIOUS
 	var/static/list/emote_list = list()
+	var/message_vox = "" // Message to display if the user is a vox sunset
```

Comment:
Comment is formatted wrong, it should be `// sunset -- Message to display if the user is a vox sunset`